### PR TITLE
[DOC-3762] [SMP-1696] Docker delegate custom certs install

### PR DESCRIPTION
**Pull request by brian-f-harness**
Docker delegate custom certs install

For reviewers, previews are available:

https://650df2d8e7943d31f9e2d201--harness-developer.netlify.app/release-notes/self-managed-enterprise-edition/#harness-delegate

https://65121409be812102dae026b1--harness-developer.netlify.app/docs/platform/delegates/secure-delegates/install-delegates-with-custom-certs#docker-delegate-self-signed-certificates-installation

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by arya-harness — APPROVED** [review]

File: docs/platform/delegates/secure-delegates/install-delegates-with-custom-certs.md [hunk 1]
```diff
@@ -364,3 +364,80 @@ spec:
                 name: my-secret-upgrader-config
 
 ```
+
+## Docker delegate self-signed certificates installation
+
+For Docker delegates, certificates are in `/opt/harness-delegate/<JDK_folder>/lib/security/cacerts`.
+
+:::info note
+The `<JDKf_folder>` location depends on the delegate.
```
Comment (pratmit) [inline docs/platform/delegates/secure-delegates/install-delegates-with-custom-certs.md:373]: JDK_folder instead of JDKf_folder

**Review by pratmit — COMMENTED** [review]
Small typo fix

File: docs/platform/delegates/secure-delegates/install-delegates-with-custom-certs.md (hunk 1 above)
Comment (brian-f-harness) [reply docs/platform/delegates/secure-delegates/install-delegates-with-custom-certs.md:373]: Thanks @pratmit

**Review by michaelcretzman — DISMISSED** [review]

**Comment by brian-f-harness** [comment]
Closing in favor of https://github.com/harness/developer-hub/pull/4142

**Pull request closed without merging**